ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -179,7 +179,13 @@ server {
     listen [::]:443 ssl http2;
     server_name foundation.nodejs.org;
 
-    return 301 https://openjsf.org/;
+    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_Nasa_FNL.pdf$         https://foundation.nodejs.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-NASA.pdf permanent;
```

Comment:
Looks like the redirect is maybe still pointing to foundation.nodejs.org?  I'll add some suggestions which point it to the direct location on openjsf.org instead.

```suggestion
    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_Nasa_FNL.pdf$         https://openjsf.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-NASA.pdf permanent;
```